package.json

```diff
@@ -14,7 +14,8 @@
   },
   "scripts": {
     "start": "node node_modules/grunt-cli/bin/grunt",
-    "test": "node node_modules/grunt-cli/bin/grunt test",
+    "test": "./node_modules/karma/bin/karma start karma.conf.js",
+    "test-dev": "node node_modules/grunt-cli/bin/grunt test",
```

Comment:
You might want @louisdante9 to look into this because he is changing these grunt stuff to gulp.